Assets/Scripts/UI/DogSlot.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Author(s): Grace Barrett-Snyder 
+ * Description: Displays a Dog's thumbnail (applies to both Outside and Home slots).
+ */
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class DogSlot : MonoBehaviour {
```

Comment:
I think we should use the MonoBehaviourExtended subclass for all MonoBehaviours. Helps w/ system unification.